src/feeds/views.py

```diff
@@ -72,6 +72,9 @@
 DEFAULT_GV_PORT = 10000
 L2_GV_PATH = '/L2'
 
+sharp_underbar_reg = re.compile('^#_+')
+sharp_underbar_numeric_reg = re.compile('#[_0-9０-９]+$')
```

Comment:
How about "hashtag" rather than "sharp"?

Reply:
I want to hear @stmtstk  opinion.